src/Core/Services/CipherService.cs

```diff
@@ -806,6 +795,21 @@ public async Task RestoreWithServerAsync(string id)
 
         // Helpers
 
+        private async Task<Tuple<SymmetricCryptoKey, EncString, SymmetricCryptoKey>> MakeAttachmentKeyAsync(string organizationId)
+        {
+            SymmetricCryptoKey attachmentKey;
+            EncString protectedAttachmentKey;
```

Comment:
⛏️ You can remove these lines and just use `var (attachmentKey, protectedAttachmentKey) = ...` afterwards so it's simpler given that you are returning on each part so you don't need an outer declaration to see if the value changes in between the flows.

Reply:
Check out [1e8ed1b](https://github.com/bitwarden/mobile/pull/2633/commits/1e8ed1b5ce8b9a67800d4130d7993eeafc7146e6) for using the `SymmetricCryptoKey` in the method 👍